web/components/feed-create.tsx

```diff
@@ -67,90 +61,3 @@ export function FeedPromo(props: { hotContracts: Contract[] }) {
     </>
   )
 }
-
-export default function FeedCreate(props: {
```

Comment:
This also seems like a nice, polished component that we might want to revive at some point. It's not as bad to kill because I think we would actually dig up this code if we needed to. So, I'm am ok to delete and decrease our maintenance burden.

Reply:
I agree with that assessment.